.github/workflows/run-all-tests.yml

```
@@ -122,39 +130,69 @@
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+      - name: Check with pyroma
+        run: |
+          python -m pip install pyroma
+          pyroma .
+      - name: Check tag name
+        if: >-
+          github.event_name == 'push' &&
+          startsWith(github.ref, 'refs/tags')
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:10: Double quote to prevent globbing and word splitting

(shellcheck)